src/Kunstmaan/NodeBundle/Form/EventListener/URLChooserFormSubscriber.php

```diff
@@ -12,6 +12,7 @@
 use Symfony\Component\Validator\Constraints\Email;
 use Symfony\Component\Validator\Constraints\Url;
 
+/** @deprecated  */
```

Comment:
Controleren of je `$attributes['choose_url'] = true;` nog nodig hebt

Reply:
ja, in de oude twig template werd deze gebruikt. deze is weer teruggezet